content/posts/redis-modules-into-arm-land-part1.md

```diff
@@ -0,0 +1,275 @@
+---
+title: "Getting Redis Modules into ARM land - Part 1"
+date: 2019-11-13
+authors:
+  - Rafi Einstein
+tags: ["redisedge", "arm"]
+---
+
+Our motivation at Redis Labs of getting into ARM land and dragging Redis Modules along was RedisEdge.

```

Comment:
Change to: "Here at Redis Labs, RedisEdge was our motivation for getting into ARM land and dragging Redis modules along."

Reply:
As another note, Style Guide says that we refer to Redis Modules as Redis modules, so lower-case every use of "modules"